notebooks/concepts/imviz_compass_mpl.ipynb

```diff
@@ -0,0 +1,578 @@
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3779ae77-97fa-49db-85e5-7e8213c0cd30",
+   "metadata": {},
+   "source": [
+    "## Case 1: Dummy WCS with 10x10 image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae33669d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = np.arange(100).reshape((10, 10))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c334e59e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = WCS()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20605e33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_ = wcs_utils.draw_compass_mpl(a, w, show=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e63db00-18a4-455c-b683-94659dc25895",
+   "metadata": {},
+   "source": [
+    "Now, let's try that in the Compass plugin. We will display this same image in Imviz. Open the Compass plugin to see the result."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f5d34d55-fe0b-4882-be37-02dededaab45",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdu = fits.ImageHDU(a, name='SCI')\n",
+    "hdu.header.update(w.to_header())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "df75a36b-0ba0-4641-9c9b-1cc4f72dd2d5",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "imviz = Imviz()\n",
+    "imviz.load_data(hdu, data_label='arr_with_dummy_wcs')\n",
+    "imviz.app"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8999c3b9-2bf3-4b6a-89e9-5f6ec843a73e",
+   "metadata": {},
+   "source": [
+    "## Case 2: Real WCS with no rotation nor distortion\n",
+    "\n",
+    "We reuse the array from above but WCS from https://learn.astropy.org/tutorials/celestial_coords1.html example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3253dd7f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = WCS({'CTYPE1': 'RA---TAN',\n",
+    "         'CUNIT1': 'deg',\n",
+    "         'CDELT1': -0.0002777777778,\n",
+    "         'CRPIX1': 1,\n",
+    "         'CRVAL1': 337.5202808,\n",
+    "         'NAXIS1': 10,\n",
+    "         'CTYPE2': 'DEC--TAN',\n",
+    "         'CUNIT2': 'deg',\n",
+    "         'CDELT2': 0.0002777777778,\n",
+    "         'CRPIX2': 1,\n",
+    "         'CRVAL2': -20.833333059999998,\n",
+    "         'NAXIS2': 10})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2e73414",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_ = wcs_utils.draw_compass_mpl(a, w)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a567e019",
+   "metadata": {},
+   "source": [
+    "Now, let's try that in the Compass plugin. We will display this same image in Imviz. Open the Compass plugin to see the result."
```

Comment:
| This PR | Ginga |
| --- | --- |
| ![dummy_wcs_learnastropy_imviz](https://user-images.githubusercontent.com/2090236/145665044-f6d7275b-36bc-4f9e-9297-885b253205a4.jpg) | ![dummy_wcs_learnastropy_ginga](https://user-images.githubusercontent.com/2090236/144470597-4da5343a-3088-4345-a2d3-6f7a11148c6c.jpg) |